### PR TITLE
Install react-intl and put IntlProvider at dashboard and storybook roots

### PR DESCRIPTION
**Pull request by tristan-orourke**
This does the bare minimum of react-intl setup so that we can start using formattedMessage.

### CODE REVIEW

File: admin/.storybook/preview.js [hunk 1]
```diff
@@ -11,7 +13,15 @@ export const parameters = {
   },
 }
 
+const messages = {en: null, fr: null};
+setIntlConfig({
+    locales: ["en", "fr"],
+    defaultLocale: "fr",
```
Comment (mnigh) [inline admin/.storybook/preview.js:19]: Any reason this is `FR` and the app [this is](url) `EN`?

**Review by mnigh — COMMENTED** [review]
Nice work! One question, otherwise ready.

File: admin/.storybook/preview.js (hunk 1 above)
Comment (tristan-orourke) [reply admin/.storybook/preview.js:19]: Oh darn, that's a mistake, left over from testing something

**Review by mnigh — APPROVED** [review]
👍

**Pull request merged**